src/lib-test/fuzzer.h

```diff
@@ -0,0 +1,16 @@
+#ifndef FUZZER_H
+#define FUZZER_H
+
+#define FUZZ_DATA(args) \
```

Comment:
should these be FUZZ_START_DATA and FUZZ_START_STR to ensure we realize to call FUZZ_END too?

Reply:
Yeah, could be clearer. Although for T_BEGIN/END consistency I used FUZZ_BEGIN_*